### PR TITLE
mass obsolete children of psychiatric disorder

### LINKED ISSUE #6695: [mass Obsolete] children of psychiatric disorder
http://purl.obolibrary.org/obo/MONDO_0002025 | psychiatric disorder | http://purl.obolibrary.org/obo/MONDO_0017241 | AP4-related intellectual disability and spastic paraplegia
-- | -- | -- | --
http://purl.obolibrary.org/obo/MONDO_0002025 | psychiatric disorder | http://purl.obolibrary.org/obo/MONDO_0016980 | ATR-X-related syndrome
http://purl.obolibrary.org/obo/MONDO_0002025 | psychiatric disorder | http://purl.obolibrary.org/obo/MONDO_0020144 | cerebrovascular dementia
http://purl.obolibrary.org/obo/MONDO_0002025 | psychiatric disorder | http://purl.obolibrary.org/obo/MONDO_0018580 | PURA-related severe neonatal hypotonia-seizures-encephalopathy syndrome
http://purl.obolibrary.org/obo/MONDO_0002025 | psychiatric disorder | http://purl.obolibrary.org/obo/MONDO_0018222 | X-linked intellectual disability due to GRIA3 anomalies

MONDO:0017241
MONDO:0016980
MONDO:0020144
MONDO:0018580
MONDO:0018222

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #6695

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -351278,17 +351261,14 @@ is_a: MONDO:0015193 {source="NCIT:C111904", source="Orphanet:364013"} ! hydrops
 
 [Term]
 id: MONDO:0018222
-name: X-linked intellectual disability due to GRIA3 anomalies
-comment: Reason of obsoletion: out of scope - MONDO:excludeGroupingClass. This is part of a large mass-obsoletion effort for ORDO terms.
-subset: obsoletion_candidate
-subset: ordo_disease {source="Orphanet:364028"}
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:351284]: remove?

**Review by twhetzel — COMMENTED** [review]
Is `ordo_disease` also a subset that should be removed? See comments inline for the 3 cases where this happens.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -335994,26 +335985,18 @@ property_value: IAO:0006012 "2023-09-01" xsd:string
 
 [Term]
 id: MONDO:0017241
-name: AP4-related intellectual disability and spastic paraplegia
-def: "A disorder that presents with spastic paraplegia and intellectual disability in which the cause of the disease is a mutation in the AP4B1 gene." [https://www.clinicalgenome.org/affiliation/40006/, PMID:29193663, PMID:29430868, PMID:30543385, PMID:321171285]
-comment: Reason of obsoletion: out of scope - MONDO:excludeGroupingClass. This is part of a large mass-obsoletione effort for ORDO terms.
-subset: obsoletion_candidate
-subset: ordo_disease {source="Orphanet:280763"}
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: yes, we want to remove these subset tags, we implemented this in this PR: https://github.com/monarch-initiative/mondo/pull/6689

**Comment by nicolevasilevsky** [comment]
I will merge this as soon as the QC passes

**Pull request merged**